lib/improver/tests/psychrometric_calculations/psychrometric_calculations/test_WetBulbTemperature.py

```diff
@@ -183,16 +179,20 @@ def test_cube_metadata(self):
         self.assertEqual(result.units, Unit('K'))
         self.assertEqual(result.name(), 'wet_bulb_temperature')
 
+    @ManageWarnings(
+        ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
```

Comment:
Do we want our basic test to be raising this warning?

Reply:
And why are so many of these warnings now being raised?

Reply:
I've now changed the lowest value away from the bottom boundary, where the float32 in-precision was causing this warning to be raised.

Reply:
Thank you for getting rid of all these warnings.